packages/web-client/app/components/common/radio-option/index.css

```diff
@@ -0,0 +1,52 @@
+.radio-option {
+  --option-opacity: 1;
+  --option-input-border-color: var(--boxel-dark);
+
+  display: grid;
+  grid-template-columns: auto 1fr 1fr;
```

Comment:
Would it be better to have just 2 columns for now?

Reply:
hmm yes, it does make sense that the shared CSS should be lowest common denominator-esque, so I’ve done that, thanks